Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -109,6 +110,37 @@ async function main(): Promise<void> {
         });
         var artifacts = [];
 
+        // Clean destination folder if requested
+        if (cleanDestinationFolder) {
```

Comment:
Could it make sense to extract this logic into a function and move it to the `file_helper.ts` module?

Reply:
Resolved